packages/block-library/src/search/index.php

```diff
@@ -43,37 +43,17 @@ function render_block_core_search( $attributes ) {
 	// Border color classes need to be applied to the elements that have a border color.
 	$border_color_classes = get_border_color_classes_for_block_core_search( $attributes );
 
-	if ( $show_label ) {
-		if ( ! empty( $attributes['label'] ) ) {
-			$label_markup = sprintf(
-				'<label for="%s" class="wp-block-search__label">%s</label>',
-				$input_id,
-				$attributes['label']
-			);
-		} else {
-			// If the label text has been removed but not toggled off, add a visually hidden label with default text.
-			$label_markup = sprintf(
-				'<label for="%s" class="wp-block-search__label screen-reader-text">%s</label>',
-				$input_id,
-				__( 'Search' )
-			);
-		}
-	} else {
-		// If there is a custom label and the label is toggled off, add a visually hidden label with custom text.
-		if ( ! empty( $attributes['label'] ) ) {
-			$label_markup = sprintf(
-				'<label for="%s" class="wp-block-search__label screen-reader-text">%s</label>',
-				$input_id,
-				$attributes['label']
-			);
-		} else {
-			// If the label text has been removed and toggled off, add a visually hidden label with default text.
-			$label_markup = sprintf(
-				'<label for="%s" class="wp-block-search__label screen-reader-text">%s</label>',
-				$input_id,
-				__( 'Search' )
-			);
-		}
+	$label_markup = sprintf(
+		'<label for="%s" class="wp-block-search__label screen-reader-text">%s</label>',
```

Comment:
```suggestion
		'<label for="%1$s" class="wp-block-search__label screen-reader-text">%2$s</label>',
```

I think there's WPCS for missing argument numbers.